scss/reset/all.scss

```diff
@@ -12,6 +12,7 @@
 @import "../utils/all";
 
 // keep this order for cascade
+@import "customProperties";
```

Comment:
If we move this customProperties file to a directory for all custom property partials, we could just import them into `utils/all`